esp/src/src-react/components/SubFiles.tsx

```diff
@@ -101,26 +102,35 @@ export const SubFiles: React.FunctionComponent<SubFilesProps> = ({
         message: nlsHPCC.RemoveSubfiles2,
         items: selection.map(item => item.Name),
         onSubmit: React.useCallback(() => {
-            WsDfu.SuperfileAction("remove", file.Name, selection, false).then(() => refresh());
-        }, [file, refresh, selection])
+            WsDfu.SuperfileAction("remove", file.Name, selection, false).then(() => {
+                refreshSubfiles();
```

Comment:
This is an async action, so the code below should be in its own `useEffect` or the "refresh" method could return a promise (somehow)?